atlas_splitter/utils.py

```diff
@@ -46,6 +47,27 @@ def create_id_generator(region_map: RegionMap) -> Iterator[int]:
     return it.count(start=last + 1)
 
 
+def id_from_acronym(region_map: RegionMap, acronym: str) -> int:
+    """Create an identifier from the input acronym.
+
+    Args:
+        region_map: map to navigate the brain region hierarchy.
+        acronym: str for the region acronym
+
+    Return:
+        region id.
+    """
+    region_id_set = region_map.find(acronym, attr="acronym", with_descendants=False)
+    if region_id_set:
+        [region_id] = region_id_set
+    else:  # acronym not present in hierarchy, generating a corresponding id
+        sha = hashlib.sha256()
+        sha.update(acronym.encode("utf-8"))
+        region_id = int(str(int(sha.hexdigest(), 16))[0:10])
```

Comment:
You can reduce the integer hash with a modulo operation:
```python
sha = hashlib.sha256(acronym.encode("utf-8"))
region_id = int(sha.hexdigest(), 16) % 10 ** 10
```

Reply:
Yes but if `int(sha.hexdigest(), 16) % 10` = 0, then `region_id` stays 0...

Reply:
Yeah, indeed. ok leave it as it is